_layouts/default.html

```diff
@@ -23,18 +23,8 @@ <h1 id="content" class="page-title">Specifying JavaScript.</h1>
       {% include intro.html %}
       {% include proposals.html %}
     </section>
+    {% include_relative _layouts/page_footer.html %}
```

Comment:
should this be in the main section?

Reply:
It's a good question. I decided to mirror the placement of the pre-existing `page_header.html` above it since it's only a single page for now. https://github.com/tc39/tc39.github.io/blob/master/_layouts/default.html#L20

For a multi-page site, I would go w/ more of a so-called &#8220;global footer&#8221; that would exist outside of the [`<main>` element](https://github.com/tc39/tc39.github.io/blob/master/_layouts/default.html#L19), which would have started at [`default.html#L27`](https://github.com/tc39/tc39.github.io/blob/master/_layouts/default.html#L27) instead.

Reply:
I don't think that, because it is a single page, that the footer should be inside the main content. The purpose of the structure of `<nav></nav><main></main><footer></footer>` isn't for single pages -- its for semantic html. This is used by accessibility technologies and also helps keep the site well organized -- regardless of if it is a single page or not. 

You can read more about that here https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML and here https://webflow.com/blog/html5-semantic-elements-and-webflow-the-essential-guide

Let's move the footer out of the main body and, if you like -- in a separate patch deal with the nav. Looking at the code  -- the nav is unnecessarily nested. There is likely cleanup there. There will probably be issues in the styling due to this nesting. That might be a nice next bug.

Reply:
It's hard to read because it isn't indented, but the page header is currently nested within the <main> element.

Reply:
I'm aware. 

If you read my comment again, I proposed moving the nav, which is nested inside of the header in a single element.

>  if you like -- in a separate patch deal with the nav. Looking at the code -- the nav is unnecessarily nested

Please move the footer out of the main section. There is no reason for it to be there.

Reply:
> [&hellip;] in a separate patch deal with the nav.

Do you think it would alright if this was taken care of in https://github.com/tc39/tc39.github.io/pull/201/commits/050462461b7aca40ef96d956eec9ed20c3ebc674?

Reply:
sgtm